pkgs/applications/window-managers/sway/default.nix

```diff
@@ -6,9 +6,8 @@
 , nixosTests
 # Used by the NixOS module:
 , isNixOS ? false
-
 , enableXWayland ? true, xorg
-, systemdSupport ? stdenv.isLinux
+, systemdSupport ? lib.meta.availableOn stdenv.hostPlatform systemd, systemd
```

Comment:
Why do we have an extra systemd here?

Reply:
Sway's relationship with systemd is.... "complicated".

In order to turn off systemd support you have to tell sway to use a package called `basu`, even if `basu` isn't available.

The whole thing is weird.

Reply:
https://github.com/NixOS/nixpkgs/pull/212876

Reply:
> Also after looking at meta.platforms this resolves to always be true.

You're forgetting `pkgsStatic`.